### ISSUE TITLE
Coverage + Lint

### ISSUE THREAD

**Issue by melissa-playfab** [couchmate/TeamCity-Phabricator-Plugin#6]
Would love to use this to solve our code coverage / lint needs. Is that still on the roadmap? There's not a great way to integrate this stuff using .net currently.

**Comment by halfmatthalfcat** [comment]
Yeah eventually. We're really strapped for time and adding features on a need-by-need basis but feel free to put in a PR if you want to add. The biggest hurdle to get this implemented would to bake in an `arc lint` command, or a linter build step, capture the output and modify the Harbormaster endpoint to emit linter messages back to Differential.
